os_site/src/_includes/post.njk

```diff
@@ -0,0 +1,39 @@
+---
+layout: base.njk
+---
+<main>
+  <article>
+    <div class="post__title t-align-center t-serif">
+        <h1>{{ title }}</h1>
+    </div>
```

Comment:
To save on the extra div, you might even get away with doing:
```
<h1 class="post__title t-align-center t-serif">{{ title }}</h1>

```

Reply:
I tried this but was unable to add the padding appropriately to the header. Let me know if there's another workaround you could see that reduces the number of div's.

Reply:
Oh yes, I see what you mean. Maybe try bottom margin on the header. `<h1 class="post__title t-align-center t-serif has-b-btm-marg">{{ title }}</h1>`

Reply:
This worked -- thank you!